content/en/blog/2024/prom-and-otel/index.md

```diff
@@ -0,0 +1,671 @@
+
+![Diagram showing the different components of the OTel Collector](OTel-collector-refresher.png)
+
+The
+[Prometheus receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/prometheusreceiver)
+allows you to collect metrics from any software that exposes Prometheus metrics.
+It serves as a drop-in replacement for Prometheus to scrape your services, and
+supports the
+[full set](https://github.com/prometheus/prometheus/blob/v2.28.1/docs/configuration/configuration.md#scrape_config)
+of configurations in `scrape_config`.
+
+If you are interested in
+[exemplars](/docs/specs/otel/metrics/data-model/#exemplars), which is a recorded
+value that associates OTel context with a metric event, you can also use the
+Prometheus receiver to ingest them in the Prometheus format and convert it to
+OTLP format. This enables you to correlate traces with metrics.
+
+Something to consider with this component is that it is under active
+development; as such, it has several
+[limitations](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/prometheusreceiver/README.md#%EF%B8%8F-warning),
+including that it’s a stateful component. Additionally, it is recommended to not
+use this component when multiple replicas of the Collector are run, because in
+this state:
+
+- The Collector is unable to auto-scale the scraping
+- If the replicas are running with the same config, it will scrape the targets
+  multiple times
+- You will need to configure each replica with a different scraping config if
+  you want to manually shard the scraping
+
+For exporting metrics from the OTel Collector to Prometheus, you have two
+options: the
+[Prometheus exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/prometheusexporter#prometheus-exporter),
+and the
+[Prometheus Remote Write exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/prometheusremotewriteexporter/README.md).
+Note that
+[Prometheus also now supports OTLP natively](https://prometheus.io/blog/2024/03/14/commitment-to-opentelemetry/).
+
+The Prometheus exporter allows you to ship data in the Prometheus format, which
+is then scraped by a Prometheus server. It's used to report metrics via the
+Prometheus scrape HTTP endpoint. You can learn more by trying out this
+[example](https://github.com/open-telemetry/opentelemetry-go/tree/main/example/prometheus).
+However, the scraping won't really scale, as all the metrics are sent in a
+single scrape.
+
+To get around the scaling concern, you can alternatively use the Prometheus
+Remote Write exporter, which allows you to push data to Prometheus from multiple
+Collector instances with no issues. Since Prometheus also accepts remote write
+ingestion, you can also use this exporter if you are generating OTel metrics and
+want to ship them to a backend that is compatible with Prometheus remote write.
```

Comment:
Might be worth mentioning that PRW in the Prometheus Server doesn't currently support metadata (help, description).  See https://github.com/prometheus/prometheus/issues/13163 and https://github.com/prometheus/prometheus/issues/12608.  This is being addressed in PRW 2.0: https://prometheus.io/docs/specs/remote_write_spec_2_0/#io-prometheus-write-v2-request

Reply:
Added this:
>Note that the Prometheus Remote Write in the Prometheus Server doesn't currently 
support metadata, such as Help and Type. For more information, check
out [issue #13163](https://github.com/prometheus/prometheus/issues/13163) as well as [issue #12608](https://github.com/prometheus/prometheus/issues/12608). This will be addressed in [Prometheus Remote Write v2.0](https://prometheus.io/docs/specs/remote_write_spec_2_0/#io-prometheus-write-v2-request).